source/includes/kubernetes_extension/_k8_pods.md

```diff
@@ -519,6 +519,67 @@ Return value:
 | `taskId` <br/>_string_     | The id corresponding to the create pod task. |
 | `taskStatus` <br/>_string_ | The status of the operation.                 |
 
+<!-------------------- REPLACE POD -------------------->
+
+#### Replace a pod
+
+```shell
+curl -X PUT \
+  -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/a_service/an_environment/pods/edgar-allen-pod/default"
+  Content-Type: application/json
+  {
+	  "apiVersion": "v1",
```

Comment:
The indentations seems to be off here as well

Reply:
Fixed formatting in 8f7df94